<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SensorMonitor</class>
 <widget class="QMainWindow" name="SensorMonitor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Meiryo UI</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>SensorMonitor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="PlotWidget" name="thrust_view"/>
        </item>
        <item row="0" column="1">
         <widget class="PlotWidget" name="pressure_view"/>
        </item>
        <item row="1" column="0">
         <widget class="PlotWidget" name="thermo_view"/>
        </item>
        <item row="1" column="1">
         <widget class="PlotWidget" name="thermo_view_2"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QDateTimeEdit" name="dateTimeEdit">
          <property name="wrapping">
           <bool>false</bool>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::ButtonSymbols::NoButtons</enum>
          </property>
          <property name="displayFormat">
           <string>yyyy/MM/dd H:mm:ss</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="thrust_label">
          <property name="frameShape">
           <enum>QFrame::Shape::Box</enum>
          </property>
          <property name="text">
           <string>thrust: 500N</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="temp_label">
          <property name="frameShape">
           <enum>QFrame::Shape::Box</enum>
          </property>
          <property name="text">
           <string>temp: 20℃</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="temp_label_2">
          <property name="frameShape">
           <enum>QFrame::Shape::Box</enum>
          </property>
          <property name="text">
           <string>temp2: 20℃</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="pressure_label">
          <property name="frameShape">
           <enum>QFrame::Shape::Box</enum>
          </property>
          <property name="text">
           <string>pressure: 1kPa</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header location="global">pyqtgraph.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
